staking/apr/compute.go

```diff
@@ -27,22 +29,21 @@ type Reader interface {
 }
 
 const (
-	secondsInYear = int64(31_557_600)
+	secondsInYear = int64(31557600)
```

Comment:
why change this line?

Reply:
it doesn't compile well in my IDE. Guess the style is not recognized universally